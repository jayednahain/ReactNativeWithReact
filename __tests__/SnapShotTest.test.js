//SnapShotTest
import 'react-native';
import React from 'react'
import MyApp from '../App/myApp';
import renderer from 'react-test-renderer';
const sum = require('../App/Utils');


// test("Sum Test ", () => { 
//     console.log("Sum Test");
//     expect(
//         sum("Jayed", "Nahian")
        
//     ).toBe("Jayed Nahian");
// });


test("Home Snap Shoot test", () => {
    const snap = renderer.create(<MyApp />).toJSON();

    expect(snap).toMatchSnapshot();

    // console.log("snap : ", snap.toJSON())
    
});
