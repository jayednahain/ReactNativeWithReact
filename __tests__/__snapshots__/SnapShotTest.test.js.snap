// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Snap Shoot test 1`] = `
<View
  style={{}}
>
  <Text>
    Show data
  </Text>
  <Text>
    Show data
  </Text>
  <Text>
    Show data
  </Text>
  <TextInput
    style={
      {
        "borderColor": "red",
        "borderWidth": 1,
      }
    }
    testID="5"
  />
</View>
`;
